### PR TITLE
Linked the pages of the developed Covid-19 News website

### PR DESCRIPTION
**Pull request by elizabethbusola**
@Cynthia-Berry @yemexx1 @michaelolubajo @Oyeteju Kindly check

### CODE REVIEW

**Comment by michaelolubajo** [comment]
@elizabethbusola Good job!
I think we need to link the read more on the homepage, as well as the images, to the corresponding news article.
Also, it appears that we've not linked to the articles themselves but to the news page. Am I wrong?
We're almost there. Well done!

**Comment by elizabethbusola** [comment]
> @elizabethbusola Good job!
> I think we need to link the read more on the homepage, as well as the images, to the corresponding news article.
> Also, it appears that we've not linked to the articles themselves but to the news page. Am I wrong?
> We're almost there. Well done!

 I actually thought about all that you mentioned but if we have to do all that then It means we will have to create more files because I strictly followed the instruction in the Hackathon task

**Comment by elizabethbusola** [comment]
> > @elizabethbusola Good job!
> > I think we need to link the read more on the homepage, as well as the images, to the corresponding news article.
> > Also, it appears that we've not linked to the articles themselves but to the news page. Am I wrong?
> > We're almost there. Well done!
> 
> I actually thought about all that you mentioned but if we have to do all that then It means we will have to create more files because I strictly followed the instruction in the Hackathon task

@michaelolubajo I have created hyperlinks for readmore, images and headlines

File: news.html [hunk 1]
```diff
@@ -98,13 +115,18 @@ <h2><a href="#">Herman Cain, US ex-presidential candidate, dies after contractin
         </article>
     </section> <br><br>
     <section>
-        <h2><a href="#">NIGERIA, IMF AND COVID-19</a></h2>
-        <b>Source:</b> CISLAC Nigeria <br>
-  		<b>Date:</b> 16th July, 2020 <br>
+        <a id="headline4">
+        <h2>NIGERIA, IMF AND COVID-19</h2>
+        <a id="fourth-image">
```
Comment (michaelolubajo) [inline news.html:120]: You didn't place the name in the image tag here?
I think the ID is supposed to be in the tag you are pointing to and not alone. Am i wrong?

**Review by michaelolubajo — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -5,43 +5,47 @@
...
-         <a href="#"><img src="images/logo.png" alt="COVIDWatch logo"></a> 
-         <a href="#">HOME</a> |
-         <a href="#">NEWS</a> |
-         <a href="#">SUBMIT NEWS</a>
-      </nav>
-      <h2><a href="#">A Roadmap to Reset the Nation's Approach to the Pandemic</a></h2>
-      <img src="images/news1.png">
+      <header>
+         <nav>
+            <a href="#"><img src="./images/logo.png" alt="COVIDWatch logo"></a>
```
Comment (nwakaemecynthia) [inline index.html:10]: This Section is properly linked. Thank you

File: news.html (hunk 1 above)
Comment (elizabethbusola) [reply news.html:120]: > You didn't place the name in the image tag here?
> I think the ID is supposed to be in the tag you are pointing to and not alone. Am i wrong?

That is how it is supposed to be. You are just supposed to place it anywhere on the webpage.

**Review by yemexx1 — APPROVED** [review]

**Pull request merged**